docs-markdown/package.json

```diff
@@ -652,9 +652,10 @@
   "scripts": {
     "vscode:prepublish": "tsc -p ./",
     "compile": "tsc -watch -p ./",
-    "postinstall": "node ./node_modules/vscode/bin/install",
-    "test-core": "node ./node_modules/vscode/bin/test",
     "test": "jest",
```

Comment:
Should the test script be switched to run mocha instead of jest?

Reply:
Fixed!